timescaledb/contribute-to-docs.md

```diff
@@ -0,0 +1,208 @@
+guidance, and in some cases differs from the other resources listed here.
+1.  The [Google Developer Documentation Style Guide][google-style]: for most
+general style guidance, we rely on the style defined here.
+1.  The Chicago Manual of Style: we use this guide for some formatting decisions
+that are not covered in other resources
+1.  Merriam-Webster: Timescale documentation is written in US English, for
+spelling and definitions, consult the dictionary.
+
+
+## Language
+We use standard US English, with an emphasis on plain (or classical) language, in simple present tense, using the second person singular ("you"). We prefer the active voice, but do not be afraid to use the passive voice if it serves a purpose. Always choose the simplest and clearest language, regardless of whether it's passive or active voice.
+
+For example, here are three ways of writing one sentence:
+
+*   Natural English: In order to perform X installation process, please ensure
+that all of the following steps are done ...
+*   Tech writer's English: To perform the X installation process, verify you have
+done the subsequent steps ...
+*   Plain English: To install X, do these steps ...
+
+Remember that the order of words is important in English. Put the most important
+part of a sentence first, this is usually the actor or the action. Use the
+second part of the sentence to give it a focus: what else should the reader
+notice?
+
+Readers are often in an agitated state by the time they get to our
+documentation. Stressed readers jump around in the text, skip words, steps, or
+paragraphs, and will quickly give up if things seem too complex. To mitigate
+this, use short sentences, plain language, and a minimum number of eye-catching
+details such as admonitions.
+
+Never assume that because you've explained something earlier in a document,
+readers will know it later in the document. You can use cross-references to help
+guide readers to further information if they need it.
+
+
+## Grammar
+Grammar rules are tacit evolving conventions. They have no implicit value by
+themselves, they only gain value because everyone is doing it.
+
+There are no hard and fast rules about dangling participles, split infinitives,
+or ending sentences with prepositions. Obeying these rules can often make
+language clearer but, in some cases, they make language more complicated. In
+that case, feel free to ignore them.
+
+
+## Verbs
+
+*   `Click` a button in a graphical user interface using a mouse. Do not `Click on`.
+*   `Press` a key or key combination on a keyboard.
+*   `Type` words or numbers using a keyboard.
+*   `Check` or `uncheck` a checkbox.
+*   `Select` or `deselect` an item in a menu.
+*   `Navigate` to a page or location in a graphical user interface.
+
+
+## Word usage
+
+### A [](#A)
+
+*Adverbs*
+: Do not use.
+
+:+1: Install TimescaleDB.
+
+:x: Simply install TimescaleDB.
+
+*And/Or*
+: Do not use. You can usually pick one. If you're not sure, pick "and".
```

Comment:
The examples below are opposite of what this says

Reply:
🤦‍♀️